global.json

```diff
@@ -1,3 +1,6 @@
 {
-    "sources": ["src"]
+    "sources": [
+        "src",
+        "submodules/FileSystem/src"
```

Comment:
Tried both `submodules/*/src` and `submodules/**/src` but none of them works. I'll file an issue for `sources` supporting globbing.